elastic/kibana
# [SecuritySolution][Onboarding] Catch completion check failure

## Summary

Fixes: https://github.com/elastic/kibana/issues/196091 | ~https://github.com/elastic/kibana/issues/196649~

1. Add the error handling when an error occurred during check complete.
2. Update the AI assistant capabilities to `[['securitySolutionAssistant.ai-assistant', 'actions.show']]`

**Steps to verify:**

https://p.elstc.co/paste/+6OYqx41#tZxjvqXgQJ2uRlCSqZH8ADMEAvR1+qnXe-5kEbt+bro
Login with the user without indices privilege. It should display the tab content without the completion information.

When completion check failed - it should display and error `toast`, regard the card as `incomplete` and `show the content`:


https://github.com/user-attachments/assets/30b1654e-99cc-4582-8beb-c4a5fb005e6f

AI assistant should not show connectors options as `add integration` is regarded as incomplete:

<img width="1761" alt="Screenshot 2024-10-17 at 15 02 42" src="https://github.com/user-attachments/assets/07fb317e-57d6-4980-aae3-7eb2d0fce12a">


Then add the index privilege:
<img width="2208" alt="Screenshot 2024-10-17 at 15 07 36" src="https://github.com/user-attachments/assets/bb879964-e31b-4ee3-8eb3-dff0381be287">

When completion check success: it should display the completion results:


https://github.com/user-attachments/assets/2a27a042-c634-4f44-bfd0-2ae503f396a2











### Checklist

Delete any items that are not applicable to this PR.
